packages/mobile/src/components/details-tile/DetailsTile.tsx

```diff
@@ -0,0 +1,354 @@
+      textTransform: 'uppercase'
+    },
+
+    coverArtWrapper: {
+      borderWidth: 1,
+      borderColor: themeColors.neutralLight8,
+      borderRadius: 4,
+      overflow: 'hidden',
+      height: 195,
+      width: 195,
+      marginBottom: 23
+    },
+
+    coverArt: {
+      borderRadius: 4,
+      overflow: 'hidden'
+    },
+
+    title: {
+      fontSize: 18,
+      textAlign: 'center',
+      marginBottom: 8
+    },
+
+    artistContainer: {
+      ...flexRowCentered(),
+      marginBottom: 16
+    },
+
+    artist: {
+      color: themeColors.secondary,
+      fontSize: 18
+    },
+
+    badge: {
+      marginLeft: 4
+    },
+
+    descriptionContainer: {
+      width: '100%'
+    },
+
+    description: {
+      fontSize: 16,
+      textAlign: 'left',
+      width: '100%',
+      marginBottom: 24
+    },
+
+    buttonSection: {
+      width: '100%',
+      marginBottom: 12
+    },
+
+    playButtonText: {
+      textTransform: 'uppercase'
+    },
+
+    infoSection: {
+      borderTopWidth: 1,
+      borderTopColor: themeColors.neutralLight7,
+      flexWrap: 'wrap',
+      flexDirection: 'row',
+      width: '100%',
+      paddingTop: 24,
+      paddingBottom: 8
+    },
+
+    noStats: {
+      borderWidth: 0
+    },
+
+    infoFact: {
+      ...flexRowCentered(),
+      flexBasis: '50%',
+      marginBottom: 16
+    },
+
+    infoLabel: {
+      ...flexRowCentered(),
+      color: themeColors.neutralLight5,
+      fontSize: 14,
+      lineHeight: 14,
+      textTransform: 'uppercase',
+      marginRight: 8
+    },
+
+    infoValue: {
+      ...flexRowCentered(),
+      lineHeight: 14,
+      color: themeColors.neutral,
+      fontSize: 14
+    },
+
+    infoIcon: {
+      marginTop: -4
+    },
+
+    link: {
+      color: themeColors.primary
+    }
+  })
+
+/**
+ * The details shown at the top of the Track Screen and Collection Screen
+ */
+export const DetailsTile = ({
+  coSign,
+  description,
+  descriptionLinkPressSource,
+  details,
+  hasReposted,
+  hasSaved,
+  hideFavorite,
+  hideFavoriteCount,
+  hideListenCount,
+  hideOverflow,
+  hideRepost,
+  hideRepostCount,
+  hideShare,
+  imageUrl,
+  onPressFavorites,
+  onPressOverflow,
+  onPressPlay,
+  onPressRepost,
+  onPressReposts,
+  onPressSave,
+  onPressShare,
+  playCount,
+  renderBottomContent,
+  renderHeader,
+  renderImage,
+  repostCount,
+  saveCount,
+  headerText,
+  title,
+  user
+}: DetailsTileProps) => {
+  const styles = useThemedStyles(createStyles)
+  const navigation = useNavigation()
+
+  const isPlaying = useSelector(getPlaying)
+  const currentUserId = useSelectorWeb(getUserId)
+
+  const isOwner = user?.user_id === currentUserId
+
+  const handlePressArtistName = useCallback(() => {
+    if (!user) {
+      return
+    }
+    navigation.push({
+      native: { screen: 'Profile', params: { handle: user.handle } },
+      web: { route: `/${user.handle}` }
+    })
+  }, [navigation, user])
+
+  const handleExternalLinkClick = useCallback(
```

Comment:
def fine for now, thinking we try and utilize a link hook or link component for some of this stuff